content/glossary/sql/alter-table.md

```diff
@@ -0,0 +1,37 @@
+<!--
+---
+Title: "ALTER TABLE"
+Subjects:
+  - "sql"
+  - "data science"
+Tags:
+  - "sql"
+  - "data science"
+  - "database"
+  - "queries"
+  - "table"
+  - "postgressql"
+  - "mysql"
+  - "sqlite"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-sql"
+  - "https://www.codecademy.com/learn/sql-table-transformation"
+  - "https://www.codecademy.com/learn/paths/design-databases-with-postgresql"
+  - "https://www.codecademy.com/learn/paths/full-stack-engineer-career-path"
+---
+-->
+
+# ALTER TABLE
```

Comment:
No need for the title in Markdown--we're covered by the Title field in the metadata!

Reply:
👍🏼